.github/workflows/openvpn.yml

```
@@ -73,7 +73,7 @@ jobs:
       - name: Test openvpn
         working-directory: ./openvpn
         run: |
-          sh ./test.sh dilithium5 p521_kyber1024
+          sh ./test.sh mldsa65 p521_mlkem1024
```

Nit: assuming we want a drop-in replacement with the same security level, this should be ML-DSA-87. There are a few similar cases throughout.

```suggestion
          sh ./test.sh mldsa87 p521_mlkem1024
```